### ISSUE TITLE
No money in .park scenarios

### ISSUE THREAD

**Issue by AdorableWolf** [OpenRCT2/OpenRCT2#16478]
### Operating System

Windows 10, 64bit

### OpenRCT2 build

OpenRCT2, v0.3.5.1-d97ecdd

### Describe the issue

I tried to search by tags "money" "is open" "dev" "multiplayer", even searched through closed ones. I might missed it as well.
I did try few of the last 30 dev builds, seems they contain this problem too.

Whenever I start scenario that is saved as .park it always is in creative, meaning I have no access to money. They work just fine in singleplayer.

### Area(s) with issue?

New Save Format (NSF), This bug is specific to multiplayer

### Steps to reproduce

1. Create/Use scenario or convert SC6 type to .park scenario on development build. 
2. Select any objective that contains money.
3. Save it. (It is now .park)
4. Run the multiplayer
5. No matter objective, there is no money
6. When you go to edit again to change the goal, it always is set to "no money".

**Comment by skyace65** [comment]
Just tested this with the latest nightly build, not seeing the issue. I started a game in singleplayer, saved it as a .park file, put it on my server, connected and money is there.

**Comment by AdorableWolf** [comment]
> Just tested this with the latest nightly build([7ead709](https://github.com/OpenRCT2/OpenRCT2/commit/7ead70968b1e834a301319df71831311d6d63467)), not seeing the issue. I started a game in singleplayer, saved it as a .park file, put it on my server, connected and money is there.

Hmm.. Any tips then what I should do? I've tried both the version you mentioned and the latest one but with no good results.
Could it be I shouldn't use automatic update(installer)? Maybe i did something wrong. I did create .park scenario without modifying anything, just making sure the money is enabled on the version you mentioned just to make sure It isn't versioning issue. Tho when I start it, it has no money. When I try to edit the saved scenario, it states that "no money" option is on instead.

**Comment by skyace65** [comment]
I don't use the launcher, I just use the portable version on Windows, but that shouldn't cause the issue. Are you having the issue with RCT1 and 2 scenarios or are you testing with scenarios you're creating yourself?

**Comment by AdorableWolf** [comment]
I have installed only RCT2 atm. But I am trying on scenarios I did download and that works fine on the old save type and just become broken after I switch those to .park. I tried to make completely clean scenario. No changes. Entrance, some space(like 10 tiles) and path. As goal I leave it as is(-> money enabled) and just save. 

I just tried it again. And no money on multiplayer and there is money in singleplayer on the same map.
